gno.land/pkg/sdk/vm/keeper.go

```diff
@@ -45,7 +46,7 @@ const (
 type VMKeeperI interface {
 	AddPackage(ctx sdk.Context, msg MsgAddPackage) error
 	Call(ctx sdk.Context, msg MsgCall) (res string, err error)
-	QueryEval(ctx sdk.Context, pkgPath string, expr string) (res string, err error)
+	QueryEval(ctx sdk.Context, pkgPath string, expr string, format Format) (res string, err error)
```

Comment:
```suggestion
	QueryEval(ctx sdk.Context, MsgQuery) (res string, err error)
```

This is where we could switch to a message struct.